src/app/zap-templates/zcl/data-model/chip/washer-controls-cluster.xml

```diff
@@ -36,7 +44,8 @@ limitations under the License.
 
     <attribute side="server" code="0x0000" define="SPIN_SPEEDS"               type="ARRAY" entryType="CHAR_STRING"                                writable="false" isNullable="false" optional="true">SpinSpeeds</attribute>
     <attribute side="server" code="0x0001" define="SPIN_SPEED_CURRENT"        type="INT8U"                                min="0x00" max="0x1F"   writable="true"  isNullable="true"  optional="true">SpinSpeedCurrent</attribute>
-    <attribute side="server" code="0x0002" define="NUMBER_OF_RINSES"          type="INT8U"                                                        writable="true"  isNullable="true"  optional="true">NumberOfRinses</attribute>
-    <attribute side="server" code="0x0003" define="MAXRINSES"                 type="INT8U"                                min="0x00" max="0x08"   writable="false"                    optional="true">MaxRinses</attribute>
+    <attribute side="server" code="0x0002" define="NUMBER_OF_RINSES"          type="NUMBER_OF_RINSES_ENUM"                                        writable="true"  isNullable="true"  optional="true">NumberOfRinses</attribute>
+    <attribute side="server" code="0x0003" define="SUPPORTED_RINSES"          type="ARRAY" entryType="NUMBER_OF_RINSES_ENUM"                      writable="true"  isNullable="true"  optional="true">SupportedRinses</attribute>
```

Comment:
NUMBER_OF_RINSES_ENUM seems like an invalid type (uses an define name somehow). Should probably be `NumberOfRinsesEnum`